docs/storage-drivers/s3.md

```diff
@@ -89,6 +89,18 @@ An implementation of the `storagedriver.StorageDriver` interface which uses Amaz
   </tr>
   <tr>
     <td>
+      <code>keyid</code>
```

Comment:
Could you also update the parameters list below with a similar entry for `keyid`?